Developer Training/docs/developer_guide.md

```diff
@@ -3112,7 +3137,7 @@ public PandemicViewTopComponent() {
 Try running Constellation now and opening your new view. Note that the
 text field will be empty as we have not told it what to display yet.
 
-![](media/image30.png)
+![](media/image32.png)
```

Comment:
is image30 used at all now or can it be removed? (Perhaps image32 should be renamed image30 to replace the old image)